### PR TITLE
Pr fix

### PR DESCRIPTION
**Pull request by aniyahb**

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# aniyahcamile@gmail.com api
+# VITE_API_KEY=5e9971f057e04bdca66fd5e64df8fd38
+
+# bunnaniyah@yahoo.com api
+VITE_API_KEY=2f4e6c77a9a744efa856550d72df9496
```
Comment (alishahimtiaz) [inline frontend/.env:5]: Don't commit API keys

File: frontend/src/RecipeCard/RecipeCard.jsx [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
-        <div className="recipeCard" onClick={props.handleClickFunc}>
+    return (
+        <div className="recipeCard" onClick={handleClick}>
+            <div className='recipeContent'>
             <section>
-                <img className ="image" src= {filler_img} />
-                <h2 className = "recipeName" > Recipe Name </h2>
+                <img className ="image" src= {props.image} />
+                <h2 className = "recipeName" > {props.title}</h2>
                 <div className='underName'>
```
Comment (alishahimtiaz) [inline frontend/src/RecipeCard/RecipeCard.jsx:21]: What is this div doing?

File: frontend/src/SignUp/SignUp.css [hunk 3]
```diff
@@ -76,14 +76,15 @@
     border:none;
     outline:none;
     color:#030303;
-    font-size: 10px;
+    font-size: 13px;
     /* right: 300px; */
```
Comment (alishahimtiaz) [inline frontend/src/SignUp/SignUp.css]: Commented out code

File: frontend/src/RecipeCard/RecipeCard.jsx [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
-import filler_img from '../assets/placeholder_img.jpeg'
+
 
 const RecipeCard = (props) => {
-    const [isFav, setIsFav] = useState(false);
+const [isFav, setIsFav] = useState(false);
+
+const key = props.id
+const handleClick = () => {
+    props.handlePickedRecipe(props.id);
```
Comment (alishahimtiaz) [inline frontend/src/RecipeCard/RecipeCard.jsx]: You can chaneg this to key

File: frontend/src/RecipeCard/RecipeCard.jsx [hunk 5]
```diff
@@ -1,18 +1,27 @@
...
+    props. setModalOpen(true);
```
Comment (alishahimtiaz) [inline frontend/src/RecipeCard/RecipeCard.jsx]: props.setModalOpen(true)

(remove space)

File: frontend/src/RecipeCard/RecipeCard.css [hunk 6]
```diff
@@ -52,15 +65,25 @@
     color:red;
 }
 
-.serving{
-    padding-left: 10px;
+.cuisine{
+    /* padding-left: 10px; */
```
Comment (alishahimtiaz) [inline frontend/src/RecipeCard/RecipeCard.css]: Commented out code

**Review by alishahimtiaz — CHANGES_REQUESTED** [review]

File: frontend/src/RecipeCard/RecipeCard.jsx (hunk 2 above)
Comment (aniyahb) [reply frontend/src/RecipeCard/RecipeCard.jsx:21]: Its just there for css purposes

**Pull request merged**